packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -274,7 +426,7 @@ describe('Sayt Component', () => {
     let hideSayt;
 
     beforeEach(() => {
-      dispatchEvent = stub(window, 'dispatchEvent');
+      dispatchEvent = stub(sayt, 'dispatchEvent');
```

Comment:
This whole test section can be removed. It's out of date and duplicated by [the describe block above](https://github.com/groupby/sfx-view/pull/31/files#diff-8340fbc81f5e03de0ad1e53a9141aa2eR240)

Reply:
@geoffcoutts Excellent catch.